### ISSUE TITLE
watson edit fails

### ISSUE THREAD

**Issue by braingineer** [TailorDev/Watson#19]
```
cogniton@ada ~/research/code/Watson $ watson edit
Traceback (most recent call last):
  File "/home/cogniton/anaconda/bin/watson", line 9, in <module>
    load_entry_point('watson==0.1', 'console_scripts', 'watson')()
  File "/home/cogniton/anaconda/lib/python2.7/site-packages/setuptools-18.1-py2.7.egg/pkg_resources/__init__.py", line 558, in load_entry_point
    
  File "/home/cogniton/anaconda/lib/python2.7/site-packages/setuptools-18.1-py2.7.egg/pkg_resources/__init__.py", line 2682, in load_entry_point
    
  File "/home/cogniton/anaconda/lib/python2.7/site-packages/setuptools-18.1-py2.7.egg/pkg_resources/__init__.py", line 2355, in load
    
  File "/home/cogniton/anaconda/lib/python2.7/site-packages/setuptools-18.1-py2.7.egg/pkg_resources/__init__.py", line 2361, in resolve
    
  File "build/bdist.linux-x86_64/egg/watson/__main__.py", line 6, in <module>
  File "/home/cogniton/anaconda/lib/python2.7/site-packages/click-5.1-py2.7.egg/click/core.py", line 700, in __call__
    return self.main(*args, **kwargs)
  File "/home/cogniton/anaconda/lib/python2.7/site-packages/click-5.1-py2.7.egg/click/core.py", line 680, in main
    rv = self.invoke(ctx)
  File "/home/cogniton/anaconda/lib/python2.7/site-packages/click-5.1-py2.7.egg/click/core.py", line 1027, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/cogniton/anaconda/lib/python2.7/site-packages/click-5.1-py2.7.egg/click/core.py", line 873, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/cogniton/anaconda/lib/python2.7/site-packages/click-5.1-py2.7.egg/click/core.py", line 508, in invoke
    return callback(*args, **kwargs)
  File "/home/cogniton/anaconda/lib/python2.7/site-packages/click-5.1-py2.7.egg/click/decorators.py", line 26, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "build/bdist.linux-x86_64/egg/watson/cli.py", line 481, in edit
  File "build/bdist.linux-x86_64/egg/watson/frames.py", line 88, in __getitem__
```

**Comment by jmaupetit** [comment]
Hi @braingineer! Thank you for the report. Do you have created any frame (via a `start` and `stop`)?

**Comment by braingineer** [comment]
Yep!  I've created one (and just tried stopping it now with success).  Another quick bug: `watson help` throws errors, `watson --help` does not.   I think your directions currently specify the former rather than the latter. 

Cheers!

**Comment by jmaupetit** [comment]
Ok now that you have created a new frame, the `edit` command works. Right?

Please report the help usage in a new issue :) Thx!

**Comment by braingineer** [comment]
re: edit

oh, yep! it lets me edit now =)

re: help.  will do =)

**Comment by jmaupetit** [comment]
Fixed since e53609b
